### ISSUE TITLE
Question about LSGAN

### ISSUE THREAD

**Issue by lanshuofeng** [hwalsuklee/tensorflow-generative-model-collections#19]
Thanks for your great work! I find that the discriminatior in LSGAN calculated by sigmoid(logits). But according to the original paper D(x) should be logits directly. If the sigmoid function is used in D(x), the problem of gradient vanishing still exists. I'm not sure whether i misinterpreted the idea of the paper.

**Comment by hwalsuklee** [comment]
Hey.

You are right. I made a mistake in using output of discriminator.
Logits must be used in calculating loss function.

I fixed corresponding code.

Thanks for your contribution.
